src/test/e2e/api/admin/metrics.e2e.test.ts

```diff
@@ -178,22 +179,21 @@ test('should save multiple projects from token', async () => {
         .expect('Content-Type', /json/)
         .expect(200);
 
-    expect(body).toMatchObject({
-        applications: [
-            {
+    expect(body.applications).toEqual(
+        expect.arrayContaining([
+            expect.objectContaining({
                 appName: 'multi-project-app',
                 usage: [
                     {
-                        environments: ['default'],
+                        environments: [DEFAULT_ENV],
                         project: 'default',
                     },
                     {
-                        environments: ['default'],
+                        environments: [DEFAULT_ENV],
                         project: 'mainProject',
                     },
                 ],
```

Comment:
nested expects? interesting. And it works?

Reply:
Yes, that's the way expect works :D